client/src/store/modules/i18n/en.js

```diff
@@ -339,5 +339,10 @@ export default {
         'Your invitees will be offered availability indefinitely into the future. ',
     YOU_CAN_CHANGE_LANGUAGE_TO: 'Your can change language to',
     DISPLAY_LANGUAGE: 'Display language',
-    MY_EVENTS: 'My Events'
+    MY_EVENTS: 'My Events',
+
+    OK: 'Ok',
+    SET_MEETING_LOCATION: 'Set meeting location',
```

Comment:
It will be great if you add all new strings to uk.js to with proper Ukrainian translation.